learn/how-tos/customize-breakpoints-for-widths-of-show-in-devices.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "How to customize breakpoints for widths of devices listed in show in devices of each widget"
+id: "customize-breakpoints-for-widths-of-show-in-devices"
+sidebar_label: "Customize breakpoints for show in devices"
+---
+---
+### Configure a widget in different devices 
+1. Drag and drop a Data Table widget on the canvas.

```

Comment:
On the top we should explain the objective: where using customisation available, a table is shown in web while same data is presented as list in different form factors.